kommunicateui/src/main/java/com/applozic/mobicomkit/uiwidgets/conversation/activity/ConversationActivity.java

```diff
@@ -681,6 +682,29 @@ void beginCrop(Uri imageUri) {
         }
     }
 
+    public int parseColorWithDefaultWhite(String color) {
+        try {
+            return Color.parseColor(color);
+        } catch (Exception invalidColorException) {
+            return Color.WHITE;
+        }
+    }
+
+    public void setToolbarTitleSubtitleColorFromSettings() {
+        if(customToolbarLayout == null) {
+            return;
+        }
+
+        int titleColor = parseColorWithDefaultWhite(alCustomizationSettings.getToolbarTitleColor());
```

Comment:
Use KmThemeHelper here. Access the colors from Theme helper.

Also fix the bug in the same file here: https://github.com/Kommunicate-io/Kommunicate-Android-Chat-SDK/blob/9851b35405f1fd3ee7b2d15db2f361d676621b01/kommunicateui/src/main/java/com/applozic/mobicomkit/uiwidgets/kommunicate/utils/KmThemeHelper.java#L62. change sentMessageBackgroundCOlor to sentMessageBorderColor

Reply:
done